### ISSUE TITLE
Experiments Question

### ISSUE THREAD

**Issue by chaoscls** [LiheYoung/ST-PlusPlus#27]
Hi! I really appreciate your jobs, that is awesome! But I' m new to SSL, and I have some questions about experiments.
1. Did you have a validation set for choosing the best model during training? Or just chose the best model with test set? 
2. I confused the meaning of "In the pseudo labeling phase, all unlabeled images are predicted with test-time augmentation, which contains five scales and horizontal filpping." mentioned in the paper, but I didn't find the corresponding code in the repo.
Look forward to your early reply, and best wish!

**Comment by LiheYoung** [comment]
1. Following other works, the validation set is directly used as the test set.
2. Please refer to https://github.com/LiheYoung/ST-PlusPlus/blob/f4ff2ebaae15fb3e2c6000b50d62a67e40058118/model/semseg/base.py#L22-L42
